CHANGELOG.md

```diff
@@ -17,7 +17,8 @@
 -----------
 
 ### Internals
-* None.
+* Add baas-network-tests nightly task for (future) testing sync client operation with non-ideal network conditions. ([PR #6852](https://github.com/realm/realm-core/pull/6852))
```

Comment:
are we not running any tests now (you mention future)?

Reply:
if it's not in master then we should update it.

Reply:
Updated